block/manager.go

```diff
@@ -328,3 +327,14 @@ func (m *Manager) setDA(daconfig string, dalcKV store.KV, logger log.Logger) err
 	m.Retriever = retriever
 	return nil
 }
+
+func (m *Manager) ValidateConsensusBlockParams() {
```

Comment:
IMO the validation here is not necessary because:
1. we already have a max protection when creating the block (i.e min(maxSize, blockBytes))
2. The valiation should be done in the sdk level when voting on gov and putting it here imo is dirty an changing the state regardless to the actual gov could be confusing

Reply:
removed validation